src/common/job/client.go

```diff
@@ -206,6 +204,9 @@ func (d *DefaultClient) PostAction(uuid, action string) error {
 				status: status,
 			}
 		}
+		if strings.Contains(err.Error(), "object is not found") {
```

Comment:
Seems that the jobservice will not return 404 error?

Reply:
Double check JS code, it will return 400 code if the error is NOT_FOUND

```go
if errs.IsObjectNotFoundError(err) {
			code = http.StatusNotFound
		} 
```